mp-jwt/src/main/java/org/apache/tomee/microprofile/jwt/config/JWTAuthConfigurationProperties.java

```diff
@@ -117,6 +119,15 @@ private JWTAuthConfiguration createJWTAuthConfiguration() {
                 config.getOptionalValue("mp.jwt.decrypt.key.algorithm", String.class).orElse(null),
                 config.getOptionalValue("mp.jwt.verify.publickey.algorithm", String.class).orElse(null));
     }
+    
+    private Boolean queryAllowExp(){
```

Comment:
```
    private Boolean queryAllowExp(){
        return config.getOptionalValue("tomee.mp.jwt.allow.no-exp", Boolean.class)
                .or(() -> config.getOptionalValue("mp.jwt.tomee.allow.no-exp", Boolean.class)
                        .map(value -> {
                            CONFIGURATION.warning("mp.jwt.tomee.allow.no-exp property is deprecated, use tomee.mp.jwt.allow.no-exp propert instead.");
                            return value;
                        }))
                .orElse(false);
    }
```

to avoid to read both entries all the time (`Config` can be slow depending the `ConfigSource`) and to avoid the `AtomicBoolean` which is not needed?

Reply:
Thanks for the improvement.

Reply:
@tichovz Can you include the feedback provided by Romain? :)